### DISCUSSION TITLE
AppleMusic and phoniebox

### DISCUSSION THREAD

**Discussion by IboDiosna** [MiczFlor/RPi-Jukebox-RFID#1869] [Ideas]
Hello,

there is a full grown apple music app for linux, apple and windows devices.
Its' called cider, refer to the link:
https://download.cider.sh/?utm_source=homepage&utm_medium=cta

I wonder, if it might be possible, to include apple music to the phonie box.

**Comment by s-martin** [comment]
Hi,
it may be possible, but at the moment someone needs to investigate and take this task.

as of today implementation of Apple Music is not planned, but new contributors and pull requests are always welcome. :)
